build.gradle

```diff
@@ -17,7 +17,8 @@ repositories {
 }
 
 dependencies {
-    compile 'com.fasterxml.jackson.core:jackson-core:2.6.4'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.10.0'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.10.0'
```

Comment:
At the time I did avoid the Jackson databind library dependency to keep the endpoint library small, the lib is 1.3 MB in size. We have some options here:

1. Keep the dependency but then we should use that API for the JSON message encoding/decoding.
2. Use the Jackson core API to encode the persistent file data too and remove the dependency.
3. Use Java Properties for the persistent data and remove databind dependency. If the data is not structured, it is sufficient.
4. Use MapDB. It could serve as general persistence layer including MQTT messages that could not be send.

I personally would go for option 4...

Reply:
Will take a look to that

Reply:
2c3e9e5 use mapdb now